### DISCUSSION TITLE
documentation engine inconvenience

### DISCUSSION THREAD

**Discussion by NesCafe62** [jedisct1/libsodium#1267] [Q&A]
When I open [https://libsodium.gitbook.io/doc/](https://libsodium.gitbook.io/doc/) and click "Search" in upper right it opens search popup. Only way to close it is to press `Escape` on keyboard. Clicking with mouse outside of popup will not close it. Even when clicking that transparent overlay area it loses focus from search bar, and then pressing `Escape` to close it stops working.

It would be better to implement:
1. Click outside feature to close search popup
2. Close popup with `Escape` keyboard key no matter what is focused on the page

**Comment by NesCafe62** [comment]
Oh I discovered that when I click overlay and hold mouse button for some time it closes popup. Just it does nothing for short clicks

**Reply by samuel-lucas6** [reply]
Alongside Frank's answer, this isn't a problem for me. It closes immediately after clicking off.

Also, I'm curious why so many people use the  .gitbook.io domain instead of doc.libsodium.org. Who is linking the .gitbook.io domain? If I search libsodium, the first thing that comes up is the doc domain, and it's the one linked on GitHub.

**Comment by jedisct1** [comment]
I didn't write the engine :)

It's [gitbook](https://www.gitbook.com).
